x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/query/LongScriptFieldExistsQuery.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.runtimefields.query;
+
+import org.elasticsearch.script.Script;
+import org.elasticsearch.xpack.runtimefields.LongScriptFieldScript;
+
+public class LongScriptFieldExistsQuery extends AbstractLongScriptFieldQuery {
+    public LongScriptFieldExistsQuery(Script script, LongScriptFieldScript.LeafFactory leafFactory, String fieldName) {
+        super(script, leafFactory, fieldName);
+    }
+
+    @Override
+    protected boolean matches(long[] values, int count) {
+        return count > 0;
+    }
+
+    @Override
+    public final String toString(String field) {
+        if (fieldName().contentEquals(field)) {
+            return "ScriptFieldExists";
```

Comment:
I would be fine with even printing the getSimpleName

Reply:
Sure!